shared/surface_flux.F90

```diff
@@ -113,12 +113,9 @@
 !! </table
 module surface_flux_mod
 
-use             fms_mod, only: close_file, mpp_pe, mpp_root_pe, write_version_number
-use             fms_mod, only: file_exist, check_nml_error, open_namelist_file, stdlog
-use   monin_obukhov_mod, only: mo_drag, mo_profile, monin_obukhov_init
-use  sat_vapor_pres_mod, only: escomp, descomp
-use       constants_mod, only: cp_air, hlv, stefan, rdgas, rvgas, grav, vonkarm
-use             mpp_mod, only: input_nml_file, FATAL, mpp_error
+use FMS, version_fms=>version, kappa_fms=>kappa, epsln_fms=>epsln
+
+use fms_io_mod, only: write_version_number, file_exist, open_namelist_file
```

Comment:
Once write_version_number is moved to fms_mod and is publicized via 'use FMS', this will need to be modified.  I believe file_exist and open_namelist_file can be replaced with the fms2_io equivalents.